spec/koans.js

```diff
@@ -384,8 +386,8 @@ describe('the JavaScript language', () => {
         j += i;
       }
 
-      //expect(i).toEqual();
-      //expect(j).toEqual();
+      expect(i).toEqual(1);
```

Comment:
este se queda comentado porque i no vive en el scope solo esta dentro de for asi que no podrias obtener el valor de i porque no esta definida